.github/workflows/Perf_Env_Build_Test_CI.yml

```diff
@@ -260,10 +260,56 @@ jobs:
           sudo docker push quay.io/${{ secrets.QAUYIO_REPOSITORY }}/${{ secrets.PACKAGE_NAME }}:latest
           echo '⌛ Wait 30 sec till image will be updated in quay.io'
           sleep 30
+          
+  grafonnet_generator:
```

Comment:
I think we need to change the job name to update_grafana_dashboard instead of grafonnet_generator